apollo-federation/src/sources/connect/validation/extended_type.rs

```diff
@@ -274,7 +304,7 @@ fn validate_field(
                 schema,
             ));
 
-            if let Some((template, coordinate)) = url_template {
+            if let Some((template, coordinate)) = url_template.clone() {
```

Comment:
Does this need to be cloned? Would `Option::as_ref` work otherwise?

Reply:
reverted this